screeps-game-api/src/js_collections/js_vec.rs

```diff
@@ -354,3 +354,51 @@ where
         }
     }
 }
+
+impl<'a, T> From<&'a [T]> for JsVec<T>
+where
+    T: JsSerialize,
+{
+    fn from(v: &'a [T]) -> Self {
+        JsVec {
+            inner: v.into(),
+            phantom: PhantomData,
```

Comment:
There is no need to specify the `'a` lifetime for the `PhantomData`?

Reply:
As far as I understand, we won't be bound at all to `'a` after this conversion?

This will send everything to JS and we'll just have a `Reference`, which can live forever, and won't be tied to the original `'a` lifetime of the data being converted.

Reply:
Then do you really need to specify the `'a` lifetime if it's not reused?